packages/Button/src/Button.js

```diff
@@ -151,6 +150,8 @@ const Button = React.forwardRef((props, ref) => {
     if (!isButtonDisabled) onClick(event);
   };
 
+  const bestTabIndex = isButtonDisabled && tabIndex === null ? -1 : tabIndex || 0;
```

Comment:
Since it's already checking if tabIndex is `null`, what is the purpose of the `|| 0`?

Reply:
Oh! That will set it to `0` as a default, instead of `null`.  `tabindex=null` is not helpful, it should be `0` by default, but I want to differentiate between `0` as a default prop, and `0` that was specified by the developer, which is why I've changed the default to `null`.